config/checkstyle-examples-suppressions.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Checkstyle//DTD SuppressionFilter Configuration 1.1//EN"
+    "https://checkstyle.org/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+  <!-- Check cannot decide between indentation 4 and 6. -->
+  <suppress checks="Indentation"
+    files="emptyforiteratorpad[\\/]Example1\.java"/>
```

Comment:
If we have issue opened for this, please do `until...` comment. If not, please open one.

Reply:
Yes, I highly recommend to format on creation of examples files. But yeah, suppression is always until issue.